tests/opwen_email_server/utils/test_email_parser.py

```diff
@@ -145,11 +169,42 @@ def assertHasCount(self, data, snippet, expected_count):
                          'Expected {} to occur {} times but got {}'.format(snippet, expected_count, actual_count))
 
     @classmethod
-    def givenTestImage(cls, content_type='image/jpeg', status=200):
-        with open(join(TEST_DATA_DIRECTORY, 'test_image.jpg'), 'rb') as image:
+    def givenTestImage(cls, content_type='image/png', status=200):
+        with open(join(TEST_DATA_DIRECTORY, 'test_image.png'), 'rb') as image:
             image_bytes = image.read()
 
-        responses.add(responses.GET, 'http://test-url.jpg',
+        responses.add(responses.GET, 'http://test-url.png',
```

Comment:
Any particular reason that this had to change from jpg to png? Would it be possible to also have a test with a jpg image to ensure that in the future we don't introduce a regression where we're able to handle png but not jpg?

Reply:
The only reason why it changed is because I had to change the size of the test image to test hte resizing function. The new larger image is png just because I thought to try a different one given the chance to change. Do you want me to add a test for all image related test cases `_with_jpeg_extension`?